libparsec/crates/crypto/src/rustcrypto/sequester.rs

```diff
@@ -327,4 +328,28 @@ impl SequesterVerifyKeyDer {
 
         Ok(data.to_vec())
     }
+
+    pub fn unsecure_unwrap(signed: &[u8]) -> Result<(&[u8], &[u8]), CryptoError> {
+        if signed.len() <= Self::ALGORITHM.len() {
+            return Err(CryptoError::Signature);
+        }
+
+        let (_algorithm, others) = signed.split_at(Self::ALGORITHM.len());
+
+        let signature_size = match others.first() {
+            // Signature: algo '@' rsa_sign_size ':' rsa_signature
+            Some(b'@') if others.len() >= 3 => {
+                1 + 2 + u16::from_le_bytes([others[1], others[2]]) as usize
+            }
```

Comment:
We need to check if `others[3] == ':'`

Reply:
`unsecure_unwrap` does not need to know if signature is good.